google-cloud-aiplatform/src/main/java/com/google/cloud/aiplatform/v1beta1/utility/ValueConverter.java

```diff
@@ -0,0 +1,48 @@
+package com.google.cloud.aiplatform.v1beta1.utility;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+
+/**
+ * Exposes utility methods for converting AI Platform messages to and
+ * from {@com.google.protobuf.Value} objects.
+ */
+public class ValueConverter {
+
+  /**
+   * An empty {@com.google.protobuf.Value} message.
+   */
+  public static final Value EMPTY_VALUE = Value.newBuilder().build();
```

Comment:
Does this need to be public? It looks like it's only used for testing (and probably belongs in the test).

Reply:
No, some of the APIs require that the customer send an empty `Value` object to the API. I added this constant here as a help to developers.